src/interface/TWSolanaAddress.cpp

```diff
@@ -0,0 +1,36 @@
+// Copyright © 2017-2020 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include <TrustWalletCore/TWSolanaAddress.h>
+#include "Solana/Address.h"
+
+using namespace TW::Solana;
+using namespace TW;
+
+struct TWSolanaAddress* _Nullable TWSolanaAddressCreateWithString(TWString* _Nonnull string) {
+    return new TWSolanaAddress{Address(TWStringUTF8Bytes(string))};
```

Comment:
there seems an extra char* <> std::string conversion, maybe just:

```cpp
auto& str = *reinterpret_cast<const std::string*>(string);
return new TWSolanaAddress{Address(str)};
```

Reply:
OK, good catch.  Maybe a TWStringStdstring could be used to encapsulate the type casts :)